 ordering food/display_order.c

```diff
@@ -0,0 +1,37 @@
+//
+// Created by q on 11/6/2019.
+//
+
+#include "display_order.h"
+#include<stdio.h>
+#include<string.h>
+
+#define MAX_FOOD_NAME 30
+#define MAX_FOOD_TYPE_NAME 30
+#define MAX_DRINK_NAME 20
+
+void displayOrderCustomerData(char username[]) {
+    printf("This is your order:\n");
+    printf("------------\n");
+    printf("Name: %s\n", username);
+}
+void displayOrderFood(char food[MAX_FOOD_NAME],int foodPrice,int *totalPrice) {
+    printf("Food items:\n");
+    printf("---%s: %d\n", food, foodPrice);
+    *totalPrice += foodPrice;
+}
+void displayOrderDrink(char drink[MAX_DRINK_NAME],int drinkPrice,int drinkChoice,int*totalPrice) {
```

Comment:
this function is doing two things... displaying but also calculating the price... try to keep the scope of each function limited to one thing